### PR TITLE
Edited select-type and event-id documents.

### PR DESCRIPTION
**Pull request by v-kikl**
@jgeurten and @jsuther1974

- select-type-of-rules-to-create: Added option 20 to table 1.
- event-id-explanations:  Added a new System Integrity Policy Options table for event ID 3099.

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 1]
```diff
@@ -86,6 +86,35 @@ To enable 3090 allow events, and 3091 and 3092 events, you must instead create a
 reg add hklm\system\currentcontrolset\control\ci -v TestFlags -t REG_DWORD -d 0x300
 ```
 
+## System Integrity Policy Options
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:89]:
```suggestion
## System Integrity Policy options
```

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 2]
```diff
@@ -70,6 +70,7 @@ You can set several rule options within a WDAC policy. Table 1 describes each ru
 | **17 Enabled:Allow Supplemental Policies** | Use this option on a base policy to allow supplemental policies to expand it. NOTE: This option is only supported on Windows 10, version 1903, and above. | No |
 | **18 Disabled:Runtime FilePath Rule Protection** | This option disables the default runtime check that only allows FilePath rules for paths that are only writable by an administrator. NOTE: This option is only supported on Windows 10, version 1903, and above. | Yes |
 | **19 Enabled:Dynamic Code Security** | Enables policy enforcement for .NET applications and dynamically loaded libraries. NOTE: This option is only supported on Windows 10, version 1803, and above. | No |
+| **20 Enabled:Revoked Expired As Unsigned** | Use this option to treat binaries signed with an expired and/or revoked certificates as "Unsigned binaries" for user mode process/components under enterprise signing scenarios. | No |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md:73]:
```suggestion
| **20 Enabled:Revoked Expired As Unsigned** | Use this option to treat binaries signed with expired and/or revoked certificates as "Unsigned binaries" for user-mode process/components under enterprise signing scenarios. | No |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-kikl, kindly commit the suggested changes. Many thanks.

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 3]
```diff
@@ -86,6 +86,35 @@ To enable 3090 allow events, and 3091 and 3092 events, you must instead create a
...
+Below are the policy options in event 3099.
```
[the remark this answers is not in the document]
Comment (jgeurten) [reply windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:90]: The table looks good. Can you please update the descriptions in each row to be consistent with spacing between the colon. Right now we have some that have spaces and others that do not

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 4]
```diff
@@ -86,6 +86,35 @@ To enable 3090 allow events, and 3091 and 3092 events, you must instead create a
...
+
+| Bit Address | Policy Rule Option |
+|-------|------|
+| 2 | Enabled:UMCI |
+| 3 | Enabled:Boot Menu Protection |
+| 4 | Enabled:Intelligent Security Graph Authorization |
+| 5 | Enabled:Invalidate EAs on Reboot |
+| 7 |Required:WHQL |
+| 8 | Enabled:Developer Dynamic Code Security |
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:99]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 5]
```diff
@@ -86,6 +86,35 @@ To enable 3090 allow events, and 3091 and 3092 events, you must instead create a
...
+| 9 | Enabled: No Revalidation Upon Refresh |
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:100]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 6]
```diff
@@ -86,6 +86,35 @@ To enable 3090 allow events, and 3091 and 3092 events, you must instead create a
...
+| 17 | Disabled:Flight Signing |
+| 18 | Enabled:Inherit Default Policy |
+| 19 | Enabled:Unsigned System Integrity Policy (Default) |
+| 20 | Enabled:Dynamic Code Security |
+| 21 | Required:EV Signers |
+| 22 | Enabled:Boot Audit on Failure |
+| 23 | Enabled:Advanced Boot Options Menu |
+| 24 | Disabled:Script Enforcement |
+| 25 | Required:Enforce Store Applications |
+| 26 | Enabled: Host Policy Enforcement |
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:114]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md (hunk 4 above)
Comment (jgeurten) [reply windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:99]: No equivalent policy rule-option

**Review by jgeurten — CHANGES_REQUESTED** [review]

**Review by jgeurten — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
@tiburd Seems to have been approved by the author. FYI. Thanks

**Comment by e0i** [comment]
@tiburd Seems to have been approved by the author. FYI. Thanks.

**Pull request merged**